core/src/main/scala/com/salesforce/op/stages/impl/selector/DefaultSelectorParams.scala

```diff
@@ -38,15 +38,15 @@ object DefaultSelectorParams {
   val MaxBin = Array(32) // bins for cont variables in trees - 32 is spark default
   val MinInstancesPerNode = Array(10, 100) // spark default 1
   val MinInfoGain = Array(0.001, 0.01, 0.1) // spark default 0
-  val Regularization = Array(0.001, 0.01, 0.1, 0.2) // spark default 0
+  val Regularization = Array(0.0001, 0.0015, 0.001, 0.01, 0.1) // spark default 0
```

Comment:
why did we make this change, are there any related experiments?

Reply:
The only related doc I can find: https://salesforce.quip.com/JhIiA1PLIYUY (streaming histogram results)